folder-aliaser.py

```diff
@@ -47,21 +47,15 @@ def get_display_name(self, path):
 		'''
 		for folder in self._folders:
 			if folder['path'] == path:
-				if 'name' in folder:
-					return folder['name']
-				else:
-					# Return the last part of the path
-					return folder['path'].split('/')[-1]
-
+				# Return the alias, or if none given return the name of the folder itself
+				return folder.get('name', folder['path'].split('/')[-1])
 		return None
 
 	def save(self, folders):
 		'''
 			Saves the given folders list to the current project 'folders' entry.
 		'''
-		projectData = self._window.project_data()
-		projectData['folders'] = folders
-		self._window.set_project_data(projectData)
+		self._window.set_project_data({**self._window.project_data(), 'folders': folders})
```

Comment:
Ah, clever!

Reply:
Finally tried this out locally, seems to throw a syntax error right at the `**`.

Reply:
Very weird.  I replied back on your separate issue.